.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Grain NEAR SDK CI Workflow
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Test
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        node-version: ["16"]
+        grain-version: ["v0.5.11"]
+
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3.1.1
+        with:
+          node-version: ${{ matrix.node-version }}
+          check-latest: true
+          cache: "npm"
+
+      # Install specific version of npm
+      - name: Setup npm 8.4.0
+        run: |
+          npm install -g npm@8.4.0
+
+      - name: Install Grain
+        uses: engineerd/configurator@v0.0.8
+        with:
+          name: grain
+          url: https://github.com/grain-lang/grain/releases/download/grain-${{ matrix.grain-version }}/grain-linux-x64
+
+      - name: Install dependencies
+        run: |
+          npm ci && cd test-suite && npm ci && cd example-contracts && npm ci
```

Comment:
lol we really need to fix the workspaces, as the top-level `npm ci` should be installing every dep.